src/plasmapy/formulary/dimensionless.py

```diff
@@ -243,10 +243,10 @@ def beta(
     The beta (:math:`β`) of a plasma is defined by
 
     .. math::
-        β = \frac{p_{th}}{p_{mag}}
+        β = \frac{p_{th}}{p_{mag}} = \frac{n k_B T}{B^2/2\mu_0}
 
-    where :math:`p_{th}` is the thermal pressure of the plasma
-    and :math:`p_{mag}` is the magnetic pressure of the plasma.
+    where :math:`p_{th}` is the '~plasmapy.formularly.misc.thermal_pressure' of the plasma
+    and :math:`p_{mag}` is the '~plasmapy.forumulary.misc.magnetic_pressure' of the plasma.
```

Comment:
```suggestion
    where :math:`p_{th}` is the `~plasmapy.formulary.misc.thermal_pressure`
    of the plasma and :math:`p_{mag}` is the
    `~plasmapy.formulary.misc.magnetic_pressure` of the plasma.
```
This changes the single quotes to backticks so that it'll create a link in the online documentation.

I had forgotten that PlasmaPy has a `thermal_pressure` function!